xgi/classes/hypergraph.py

```diff
@@ -512,10 +513,20 @@ def add_edges_from(self, ebunch_to_add, **attr):
 
         Parameters
         ----------
-        ebunch_to_add : iterable of edges
-            Each edge given in the iterable will be added to the
-            graph. Each edge must be given as as an iterable of nodes
-            or an iterable with the last entry as a dictionary.
+        ebunch_to_add : iterable
+
+            An iterable of edges, in one of the following formats.
+
+            * Format 1: Iterable of hashables (id1, id2, ...), or
+            * Format 2: 2-tuple (members, id), or
+            * Format 3: 2-tuple (members, attr), or
+            * Format 4: 3-tuple (members, id, attr),
+
+            where `members` is an iterable of node IDs, each `id` is a hashable to use
```

Comment:
and here 
"where `members` is an iterable of node IDs (just like format 1), each `id` is a hashable to use..."

I would maybe even use `edge_id` in formats 2 and 4 to make it clear

Reply:
Done!